googletest/CMakeLists.txt

```diff
@@ -13,13 +13,24 @@ FetchContent_MakeAvailable(googletest)
 list(APPEND dxlibs "")
 if(EXISTS "/dev/dxg" AND EXISTS "/lib/wsl/lib/")
   find_library(libd3d12 d3d12 HINTS /lib/wsl/lib)
-  find_library(libdxcore dxcore HINTS /lib/wsl/lib)
-  list(APPEND dxlibs ${libd3d12} ${libdxcore})
+  list(APPEND dxlibs ${libd3d12})
 else()
 # Fallback to default: let CMake look for libs
-  list(APPEND dxlibs d3d12 dxcore)
```

Comment:
This test doesn't need ``dxcore`` to link. This is important because MinGW's integrated libraires do not yet include dxcore.